gateway.py

```diff
@@ -54,7 +54,11 @@ def __call__(self, json_response, binary_components):
             except StopIteration:
                 logger.error('Ran out of binary components for JSON part {}'.format(i))
             
-            if binary_type in {'probability_mask'}:
+            if binary_type in {'dicom_secondary_capture'}:
+                # Binary blob is assumed to be a file pointer or buffer type
+                # to be read directly into the response
+                yield ('application/dicom', binary_blob)
```

Comment:
I am not sure about this. What do you think @bsblaw, is `application/dicom` correct?

Reply:
application/dicom is correct. That's what we use in in our multiple request as well.